### ISSUE TITLE
Problem with rails 7.0.2.3 during bundle install

### ISSUE THREAD

**Issue by gbisheimer** [gbarillot/rails-vue-demo-app#68]
Hi. I've just downloaded your repo and wanted to try it out.

After cloning it, I installed ruby-2.6.5 (I'm using RVM) and bundler 2.3.12 to use the same version of everything.

When I ran `bundle install`, I got the following error:

`Your bundle is locked to rails (7.0.2.3) from rubygems repository https://rubygems.org/ or installed locally, but that version can no longer be found in that source. That means the author of rails (7.0.2.3) ha
removed it. You'll need to update your bundle to a version other than rails (7.0.2.3) that hasn't been removed in order to install.`

![image](https://user-images.githubusercontent.com/3077123/167912627-cd45e305-1b34-4f0d-a7a7-22d8763b36b8.png)

Updating rails to version 7.0.2.4 is not possible as it requires ruby >= 2.7. So I modified .ruby-version to use ruby-2.7.2, installed that version using RVM and then run bundle install again. Everything works just fine now.

Thanks for the repo.

**Comment by gbarillot** [comment]
Hi,

Right, for some reason the Rails version was too strict in the Gemfile. Try again with the latest version (just comment/remove the version number along the Rails dependency in the Gemfile).

Anyway, I updated the Gemfile in order to use any Rails version between 7.0.2 and 7.1.0

**Comment by gbisheimer** [comment]
I'm still having the same issue with the updated gemfile.

**Comment by junjizhi** [comment]
I got the following issue:

```
$ bundle
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...........................
Resolving dependencies............
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    rails (~> 7.0.2) was resolved to 7.0.2.3, which depends on
      Ruby (>= 2.7)

  Current Ruby version:
    Ruby (= 2.6.5)
```

**Comment by gbarillot** [comment]
Please install Ruby > 2.7 (or better > 3.0) before "bundle install". Bundler and Gemfile handle versionning and installation for everything except for Ruby, that you have to take install your own.
